core/src/main/web/app/template/index_template.html

```diff
@@ -109,6 +109,12 @@
   <script src="vendor/bower_components/datetimepicker/jquery.datetimepicker.js"></script>
   <!-- endbuild -->
 
+  <!-- htmlbuild:embedremove -->
+  <script>
+    (function(w){ w.rAF = requestFrame('request'); })(window);
```

Comment:
i think you right, @tomca32 
i checked once again: the use of requestFrame('native') + window.requestAnimationFrame is the equivalent of how it worked when we used rAf.js
i will make the necessary changes